lib/from_honeybee/geometry/face.rb

```diff
@@ -63,10 +63,9 @@ def to_openstudio(openstudio_model)
       end
       reordered_vertices = OpenStudio.reorderULC(os_vertices)
 
-      os_surface = OpenStudio::Model::Surface.new(reordered_vertices, openstudio_model)        
+      os_surface = OpenStudio::Model::Surface.new(reordered_vertices, openstudio_model)     
```

Comment:
@chriswmackey I am not sure why it changed to the earlier reordered_vertices statements. Maybe when I rebased my fork .